interface/client/mistAPIBackend.js

```diff
@@ -17,15 +19,7 @@ var filterId = function(str) {
 };
 
 
-var sound = {
-    bip: document.createElement('audio'),
-    bloop: document.createElement('audio'),
-    invite: document.createElement('audio'),
-};
-sound.bip.src = 'file://'+ dirname + '/../../sounds/bip.mp3';
```

Comment:
I did it like this, so that the sounds are preloaded. With your approach it would load the file in the moment it uses them, which could lead to delays, i guess.